### DISCUSSION TITLE
Runtime configuration

### DISCUSSION THREAD

**Discussion by shnizzedy** [FCP-INDI/C-PAC#1438] [Resolved] [answered]
This caught my eye when working on #1437: https://github.com/FCP-INDI/C-PAC/blob/1cbbdaf09ea6a1e90392ad00c4e18853e3118414/dev/docker_data/run.py#L486

I suppose we should sometime update these kinds of runtime configurations (not in the config file) to
* fit into the nested structure (like this one I'd think would end up being something like `['pipeline_setup']['Amazon-AWS']['aws_input_bucket_credentials']`
* validate the keys/values

**Comment by sgiavasis** [comment, marked as answer]
Absolutely - I would even fold this into 1.8.1 or 1.8.2.

**Reply by sgiavasis** [reply]
@shnizzedy Actually can we draft this as an Issue? Would be good to get it documented as an action item.

**Reply by shnizzedy** [reply]
#1468
